caddy/Caddyfile

```diff
@@ -1,22 +1,12 @@
 demo.immich.app {
-	@update-config {
-		method PUT
+	@blocked-paths {
+		method PUT POST
 		path /api/system-config
-	}
-
-	@update-user {
-		method PUT
-		path /api/user
-	}
-
-	@set-profile-image {
-		method POST
-		path /api/user/profile-image
-	}
-
-	@upload-file {
-		method POST
+		path /api/users/me
```

Comment:
Probably /api/user/* is fine. No put/post methods should be valid.

Reply:
My thinking was people might want to create other users to play with partner sharing and such.

Reply:
Creating users is POST /admin/users

Reply:
Although it is a bit pointless to create other users if they can't upload anything